### DISCUSSION TITLE
[Plugin request] AI chat bot

### DISCUSSION THREAD

**Discussion by nikolicpetar17** [Flow-Launcher/Flow.Launcher#1973] [Plugin Request]
Since ChatGPT api is free for commercial use I tought that it would be cool if there was plugin that allowed you to talk to ChatGPT directly through Flow Launcher. Will this be possible in the future?

**Comment by deefrawley** [comment]
We have had a few requests for this. Given the limited space of search query responses in Flow it is not practical to chat back and forth with an AI chatbot in Flow. The best you could do was launch a URL with a search request to the AI website which you can do now through the web search plugin. This would then open a browser where you would continue the conversation.

**Comment by deefrawley** [comment]
@nikolicpetar17 I have moved this to our plugin request discussion area so others can comment with ideas.

**Comment by jjw24** [comment]
@nikolicpetar17 A ChatGPT plugin has been made and available in the plugin store now.

**Comment by oeloo** [comment]
Does the use of ChatGPT plugin requires to pay the subscription ?
I have this message when using the Flow Launcher ChatGPT plugin:
`An error occurred 
You exceeded your current quota, please check your plan and billing detail...`

It was my 1st ChatGPT search, so no quota was exceeded. Unless it is only for ChatGPT paid users. Sadly there is no mention about this the ChatGPT plugin for Flow Launcher support page.

**Reply by deefrawley** [reply]
Did you enter in your own API key as per the plugin instructions?

**Reply by oeloo** [reply]
Yes, I entered the ChatGPT API key insite ChatGPT plugin for FL. I have seen there are dozen of people reporting the same issue on the support page.

**Reply by Garulf** [reply]
You only get $5 credit for the first 3 months. After which OpenAI is a paid service. 

https://openai.com/pricing
